config/proto/targets.proto

```diff
@@ -27,22 +40,24 @@ message TargetsDef {
 
   TargetSystem target_system = 1;
   ConnectionType connection_type = 2;
-  ConnectionDetails connection_details = 3;
+  CephConnectionDetails ceph_connection_details = 3;
```

Comment:
Could you just make this a connection details? All the contents look like things that should be required for most probes, and. then you could reuse this for other storage systems in future?

Reply:
I've amended this to be ConnectionDetails. We had changed it to CephConnectionDetails as GCS won't need this, but it does make sense that other storage systems will require this in the future.